README.md

```diff
@@ -1,2 +1,61 @@
 # next-runtime-env
-NextJS plugin to pass environment variables to NextJS's runtime configuration
+NextJS plugin to pass environment variables to NextJS's runtime configuration.
+
+By default, NextJS does not make available `process.env` to the client side. The available solution for this problem is to configure a runtime config in `next.config.js` to pass values to either server and client or just server.
+
+The way to do it is by passing values to the `publicRuntimeConfig` and `serverRuntimeConfig` properties of the NextJS' config.
+
+This plugin does just that.
+
+## Installation
+
+```sh
+$ npm i -D @moxy/next-runtime-env
+```
+
+This library is written and published in modern JavaScript. If you target older versions of node please make sure to transpile accordingly.
+
+## Usage
+
+```js
+// next.config.js
+const withRuntimeEnv = require('@moxy/next-runtime-env');
+
+module.exports = withRuntimeEnv();
+```
+
+Multiple configurations can be combined together with function composition. For example:
+
+```js
+// next.config.js
+const withCSS = require('@zeit/next-css');
+const withRuntimeEnv = require('@moxy/next-runtime-env');
+
+module.exports = withCSS(
+    withRuntimeEnv()({
+        cssModules: true, // this options will be passed to withCSS plugin through nextConfig
```

Comment:
Passed some options and also change the above example since it was incorrect.

Using

```js
module.exports = withRuntimeEnv();
```

was incorrect because NextJS calls this file with `(phase, nextConfig)` and not with `(nextConfig)`. Because of this, when using only this plugin, we have to use like this:

```js
module.exports = withRuntimeEnv()();
```

However this means we will be using an empty `nextConfig`. This is not problematic since we're not using anything from the default configuration. However, a more correct way of using is:

```js
module.exports = (phase, nextConfig) => withRuntimeEnv({ ...options })(nextConfig);
```

But that is strange for a single plugin.